File: src/backend/diary-server/src/main/java/com/spuit/maum/diaryserver/web/response/Diary/DiaryDetailResponse.java
package com.spuit.maum.diaryserver.web.response.Diary;

import com.spuit.maum.diaryserver.domain.diary.Emotion;
import com.spuit.maum.diaryserver.domain.diary.Music;
import java.time.LocalDate;
import java.util.List;
import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DiaryDetailResponse {

  LocalDate date;
  Emotion emotions;
  String subject;
  String content;
  List<Music> musicList;

}

File: src/frontend/Maum/app/src/main/java/com/spuit/maum/dto/DiaryDetailResponse.kt
package com.spuit.maum.dto

import java.io.Serializable

/**
 * 일기 상세 정보 Response
 */
class DiaryDetailResponse(
    var date: String? = null,
    var emotions: DetailEmotions? = null,
    var subject: String? = null,
    var content: String? = null,
    var musicList: ArrayList<Music>? = null
) : Serializable